tiatoolbox/cli/visualize.py

```diff
@@ -0,0 +1,127 @@
+"""Command line interface for visualization tool."""
+from __future__ import annotations
+
+import os
+import subprocess
+import sys
+from pathlib import Path
+from threading import Thread
+
+import click
+
+if sys.version_info >= (3, 9):  # pragma: no cover
+    import importlib.resources as importlib_resources
+else:  # pragma: no cover
+    # To support Python 3.8
+    import importlib_resources  # type: ignore[import-not-found]
+from flask_cors import CORS
+
+from tiatoolbox.cli.common import tiatoolbox_cli
+from tiatoolbox.visualization.tileserver import TileServer
+
+BOKEH_PATH = importlib_resources.files("tiatoolbox.visualization.bokeh_app")
+
+
+def run_tileserver() -> None:
+    """Helper function to launch a tileserver."""
+
+    def run_app() -> None:
+        """Run the tileserver app."""
+        app = TileServer(
+            title="Tiatoolbox TileServer",
+            layers={},
+        )
+        CORS(app, send_wildcard=True)
+        app.run(host="127.0.0.1", threaded=False)
+
+    proc = Thread(target=run_app, daemon=True)
+    proc.start()
+
+
+def run_bokeh(img_input: list[str], port: int, *, noshow: bool) -> None:
+    """Start the bokeh server."""
+    cmd = [
+        "bokeh",
+        "serve",
+    ]
+    if not noshow:
+        cmd = [*cmd, "--show"]  # pragma: no cover
+    cmd = [
+        *cmd,
+        BOKEH_PATH,
+        "--port",
+        str(port),
+        "--unused-session-lifetime",
+        "1000",
+        "--check-unused-sessions",
+        "1000",
+        "--args",
+        *img_input,
+    ]
+    subprocess.run(cmd, check=True, cwd=str(Path.cwd()), env=os.environ)  # noqa: S603
+
+
+@tiatoolbox_cli.command()
+@click.option(
+    "--base-path",
+    help="""Path to base directory containing images to be displayed.
+    Slides and overlays to be visualized are expected in subdirectories of the
+    base directory named slides and overlays, respectively. It is also possible
+    to provide a slide and overlay path separately
+    (use --slide-path and --overlay-path).""",
+)
+@click.option(
+    "--slide-path",
+    help="""Path to directory containing slides to be displayed.
+    This option must be used in conjunction with --overlay-path.
+    The --base-path option should not be used in this case.""",
+)
+@click.option(
+    "--overlay-path",
+    help="""Path to directory containing overlays to be displayed.
+    This option must be used in conjunction with --slide-path.
+    The --base-path option should not be used in this case.""",
+)
```

Comment:
I find these options a bit messy. Why have a base path in addition to a slide path and an overlay path? Why not just use positional args and treat everything as a layer? E.g.
```
$ tiatoolbox visualize slide1.svs overlay1.png
```
This would also allow for globbing e.g. `tiatoolbox visualise *.svs` etc. I understand that the CLI generally does not use positional arguments, so perhaps this could be cleaned up in a later PR. If you want to seperate images and overlays you could still do this e.g. `tiatoolbox visualize --slides *.svs --overlays *.png` etc.

Reply:
My main gripe is that there should be ONE obvious way of doing it: https://peps.python.org/pep-0020/

Reply:
I think this would be really useful `tiatoolbox visualize --slides *.svs --overlays *.png`